### ISSUE TITLE
Support for aria-hidden=true

### ISSUE THREAD

**Issue by MaartenWL** [MaartenWL/accessible-email.org#3]
implementation in all checks. Elements containing aria-hidden=true should be ignored, just like elements containing role="presentation".

**Comment by MaartenWL** [comment]
implemented
